go/vt/schemadiff/schema.go

```diff
@@ -750,3 +759,258 @@ func (s *Schema) Apply(diffs []EntityDiff) (*Schema, error) {
 	}
 	return dup, nil
 }
+
+// TODO
+
+func (s *Schema) ValidateViewReferences() error {
+	var allerrors error
```

Comment:
Usually use `errs` as a the name for this kinda thing to collect errors.

Reply:
fixed